lib/quiet_quality/config/finder.rb

```diff
@@ -0,0 +1,45 @@
+module QuietQuality
+  module Config
+    class Finder
+      CONFIG_FILENAME = ".quiet_quality.yml"
+      MAXIMUM_SEARCH_DEPTH = 100
```

Comment:
This seems really far to search up the tree. I suppose it won't _hurt_ anything and is simply a limit to ensure against trouble.